app/_src/gateway/breaking-changes/32x.md

```diff
@@ -142,4 +142,16 @@ Old parameter name | New parameter name
 `session_cookie_renew` | Removed, no replacement parameter. 
 `session_cookie_maxsize` | Removed, no replacement parameter. 
 `session_strategy` | Removed, no replacement parameter. 
-`session_compressor` | Removed, no replacement parameter. 
\ No newline at end of file
+`session_compressor` | Removed, no replacement parameter. 
+
+## Patch release breaking changes
+
+#### 3.2.2.4
+* **Ubuntu 18.04 support removed**: Support for running Kong Gateway on Ubuntu 18.04 ("Bionic") is now deprecated,
```

Comment:
You'll want to move "vale off" to the top of the file, and don't apply these variables. We don't use variables in the changelog since we don't want to change the product name here, if it ever needs changing - it's a snapshot in time.

Reply:
@lena-larionova This Vale error is on the new breaking changes page, which is why I used the variable.

Reply:
🤦‍♀️ yep, yep you're right. I totally missed that.